React-Homeworks/src/components/Buttons/Buttons.jsx

```diff
@@ -0,0 +1,13 @@
+// Buttons.js
+import React from 'react';
+import "./Buttons.css";
+
+function Button({ buttonInfo, onClick, className }) {
```

Comment:
As we discussed in the lecture it's better pass parameters to define the button view then css classes